api/repos.go

```diff
@@ -139,6 +149,10 @@ func apiReposDrop(c *gin.Context) {
 		return
 	}
 
+	if !checkGroup(repo.LdapGroup) {
```

Comment:
Given that this "if" check is repeated identical multiple times, I would refactor it.

Reply:
has been :)